tests/test_elastic_lammps_functional.py

```diff
@@ -0,0 +1,117 @@
+import os
+
+from ase.build import bulk
+import numpy as np
+import unittest
+
+from atomistics.workflows.elastic.workflow import (
+    analyse_structures_helper,
+    generate_structures_helper
+)
+
+try:
+    from atomistics.calculators import (
+        evaluate_with_lammps, get_potential_by_name
+    )
+
+    skip_lammps_test = False
+except ImportError:
+    skip_lammps_test = True
+
+
+@unittest.skipIf(
+    skip_lammps_test, "LAMMPS is not installed, so the LAMMPS tests are skipped."
+)
+class TestElastic(unittest.TestCase):
+    def test_calc_elastic_functions(self):
+        structure = bulk("Al", cubic=True)
+        df_pot_selected = get_potential_by_name(
+            potential_name='1999--Mishin-Y--Al--LAMMPS--ipr1',
+            resource_path=os.path.join(os.path.dirname(__file__), "static", "lammps"),
+        )
+        result_dict = evaluate_with_lammps(
+            task_dict={"optimize_positions_and_volume": structure},
+            potential_dataframe=df_pot_selected,
+        )
+        sym_dict, structure_dict = generate_structures_helper(
+            structure=result_dict["structure_with_optimized_positions_and_volume"],
+            eps_range=0.005,
+            num_of_point=5,
+            zero_strain_job_name="s_e_0",
+            sqrt_eta=True,
+        )
+        result_dict = evaluate_with_lammps(
+            task_dict={"calc_energy": structure_dict},
+            potential_dataframe=df_pot_selected,
+        )
+        sym_dict, elastic_dict = analyse_structures_helper(
+            output_dict=result_dict,
+            sym_dict=sym_dict,
+            fit_order=2,
+            zero_strain_job_name="s_e_0",
+        )
+        self.assertTrue(np.isclose(elastic_dict["elastic_matrix"], np.array(
+            [
+                [114.10311701, 60.51102935, 60.51102935, 0., 0., 0.],
+                [60.51102935, 114.10311701, 60.51102935, 0., 0., 0.],
+                [60.51102935, 60.51102935, 114.10311701, 0., 0., 0.],
+                [0., 0., 0., 51.23853765, 0., 0.],
+                [0., 0., 0., 0., 51.23853765, 0.],
+                [0., 0., 0., 0., 0., 51.23853765]
+            ])
+        ).all())
+        self.assertEqual(sym_dict['SGN'], 225)
+        self.assertEqual(sym_dict['LC'], 'CI')
+        self.assertEqual(sym_dict['Lag_strain_list'], ['01', '08', '23'])
+        self.assertTrue(np.isclose(sym_dict['epss'], np.array([-0.005, -0.0025, 0., 0.0025, 0.005])).all())
+        self.assertTrue(np.isclose(sym_dict["v0"], 66.43035441556098))
```

Comment:
If the dict value is as scalar as it looks you can cut numpy and directly use `assertAlmostEqual`

Reply:
Fixed, I also applied it to the other tests in https://github.com/pyiron/atomistics/pull/251